src/components/progress.scss

```diff
@@ -0,0 +1,133 @@
+// * Progress Bar component
+// *
+// *
+
+.ProgressBar-range {
+  height: $su-small;
+  flex: 1; // for inline style
+  background: $cu-middleground--dark;
+  box-shadow: inset $box-shadow-small;
+  border-radius: $border-radius-large;
+  margin-bottom: $su-small;
+  .ProgressBar--large & {
+    height: $su-default;
+  }
+  .ProgressBar--small & {
+    height: $su-xsmall;
+  }
+  .ProgressBar--extra-small & {
+    height: $su-xxsmall;
+  }
+  .ProgressBar--extra-large & {
+    height: $su-large;
+  }
+}
+.ProgressBar-title {
+  margin-bottom: $su-xsmall;
+}
+
+.ProgressBar-status {
+  background: $cu-positive;
+  height: 100%;
+  border-radius: $border-radius-large;
+  &.ProgressBar-status--canceled,
+  &.ProgressBar-status--failed,
+  &.ProgressBar-status--danger {
```

Comment:
Could we reduce this to one class? Like `ProgressBar-negative`

Reply:
These classes were already in the component, so I just used them. If it's okay to drop the others, I am all for it!

Reply:
Yeah, see comment from when Alan was working on this: https://github.com/teamsnap/teamsnap-ui/pull/25#pullrequestreview-49813369

Reply:
Whoops that link didn't work as expected... try this: https://github.com/teamsnap/teamsnap-ui/pull/25#discussion_r127248001

Reply:
Sounds good. In that case, do we need "ProgressBar--positive?" It seems from the style guide that the positive color is the default.

Reply:
Sorry -- missed this comment. @williamdingwall might be better able to answer this question.

Reply:
No we don't think we do since it's the default.  By default all progress within are system is positive. Unless specified with one of the two modifiers, `ProgressBar--negative` and `ProgressBar--neutral`